diff -up codelite-16.0.0/Runtime/codelite.desktop.1~ codelite-16.0.0/Runtime/codelite.desktop
--- codelite-16.0.0/Runtime/codelite.desktop.1~	2022-09-26 01:57:09.483545322 +0200
+++ codelite-16.0.0/Runtime/codelite.desktop	2022-09-26 02:02:36.812536870 +0200
@@ -1,5 +1,7 @@
 [Desktop Entry]
 Name=CodeLite
+Comment=IDE for the C/C++ programming languages
+Comment[ru]=Свободная, кроссплатформенная среда разработки программного обеспечения для языков С/С++ с открытым исходным кодом
 Exec=codelite %f
 Icon=codelite
 Terminal=false
