--- configure.link	2012-03-03 19:04:22.652654950 +0100
+++ configure	2012-03-03 19:04:56.563609036 +0100
@@ -517,7 +517,7 @@ createRules_sharedObject ()
 		echo "${rule_name}: \$(${object_list_name})" >> Makefile
 	fi
 	
-	echo "	\$(CMP) -fPIC ${shared_obj_switch} -o ${base_dir}/\$(OUTPUT_DIR)/${libname}.so \$(${object_list_name}) \$(PLUGIN_LINK_FLAGS) ${additional_link_libs} -lutil" >> Makefile
+	echo "	\$(CMP) -fPIC ${shared_obj_switch} -o ${base_dir}/\$(OUTPUT_DIR)/${libname}.so \$(${object_list_name}) \$(PLUGIN_LINK_FLAGS) ${additional_link_libs} -lutil -ldl" >> Makefile
 	if [ $dont_copy != "dont_copy" ]; then
 		echo "	cp -f ${base_dir}/\$(OUTPUT_DIR)/${libname}.so ${runtime_path}/." >> Makefile
 	fi
